### PR TITLE
remove duplicate calls to initObjectManager in bootstrap class

### PR DESCRIPTION
**Pull request by sivajik34**
There are duplicate calls for initobjectManager method,i think we can initialize objectmanager in constructor,so that object manager class will be available at the time of bootstrap object creation.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=9140) <br/>All committers have signed the CLA.

File: lib/internal/Magento/Framework/App/Bootstrap.php [hunk 1]
```diff
@@ -279,7 +279,6 @@ protected function assertMaintenance()
         if (null === $isExpected) {
             return;
         }
-        $this->initObjectManager();
         /** @var \Magento\Framework\App\MaintenanceMode $maintenance */
         $this->maintenance = $this->objectManager->get(\Magento\Framework\App\MaintenanceMode::class);
```
Comment (Bartlomiejsz) [inline lib/internal/Magento/Framework/App/Bootstrap.php:282]: I think this could also be removed :) Maintenance is now also initialized in constructor.

Comment (sivajik34) [reply lib/internal/Magento/Framework/App/Bootstrap.php:282]: @Bartlomiejsz  i think we can remove  Maintenance initialization  from constructor .
I don't know why Maintenance initialization happens in two times (assertMaintenance method,initObjectManager method)
i think Its enough in assertMaintenance.
anyway I'm looking for core team feedback?

Comment (ishakhsuvarov) [reply lib/internal/Magento/Framework/App/Bootstrap.php:282]: @sivajik34 @Bartlomiejsz To me it looks like it is enough only once. Seems to be some artifacts from legacy code. Going to look into it more deeply.

**Comment by okorshenko** [comment]
@sivajik34 thank you for your contribution!

**Pull request merged**